waku/v2/protocol/waku_rln_relay/waku_rln_relay_types.nim

```diff
@@ -87,6 +91,10 @@ type WakuRLNRelay* = ref object
 type MessageValidationResult* {.pure.} = enum
   Valid, Invalid, Spam
 
+# RLN membership key and index files path
+const
+  RLN_CREDENTIALS_FILEPATH* = "rlnCredentials.txt"
```

Comment:
Can we pass this path to the "waku_rln_relay" module through the node configuration?

Using hardcoded paths is a bad practice in general. And if at some point we want to support Windows, this may bite us in the future 😁 🐺

Reply:
@LNSD Thanks for your comments. This method of persistence is only a placeholder until a proper keystore and wallet is integrated. We can open this feature as a new issue if you want.